src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedFieldSymbolBase.cs

```diff
@@ -86,6 +86,10 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
 
         internal abstract override TypeWithAnnotations GetFieldType(ConsList<FieldSymbol> fieldsBeingBound);
 
+        public sealed override RefKind RefKind => RefKind.None;
```

Comment:
>public sealed override RefKind RefKind => RefKind.None; [](http://example.com/codeflow?start=8&length=55)

Is it safe to provide this implementation across all current and future derived types?